demo/react-native/ios/PorcupineDemo/AppDelegate.m

```diff
@@ -4,6 +4,10 @@
 #import <React/RCTBundleURLProvider.h>
 #import <React/RCTRootView.h>
 
+#if RCT_DEV
```

Comment:
what's this for??

Reply:
This was to remove a warning for dev environment when running iOS demo.